include/psa/crypto_values.h

```diff
@@ -1341,6 +1387,24 @@
 #define PSA_ALG_IS_ECDH(alg) \
     (PSA_ALG_KEY_AGREEMENT_GET_BASE(alg) == PSA_ALG_ECDH_BASE)
 
+/** Whether the specified algorithm encoding is a wildcard.
+ *
+ * Wildcard values may only be used to set the usage algorithm field in
+ * a policy, not to perform an operation.
+ *
+ * \param alg An algorithm identifier (value of type #psa_algorithm_t).
+ *
+ * \return 1 if \c alg is a wildcard algorithm encoding.
+ * \return 0 if \c alg is a non-wildcard algorithm encoding (suitable for
+ *         an operation).
+ * \return This macro may return either 0 or 1 if \c alg is not a supported
+ *         algorithm identifier.
+ */
+#define PSA_ALG_IS_WILDCARD(alg)                        \
```

Comment:
Yes it would, I was thinking of that in the first place, I was just trying to go for something shorter that captures the same meaning.

Reply:
“Suitable for an operation” might include “supported” and this can't be tested statically. I can't think of a semantics that could be implemented other than “does not use a wildcard”, and `IS_NOT_WILDCARD` is best presented as `!IS_WILDCARD`.

Reply:
Oh I see, I haven't thought of that.